### ISSUE TITLE
Discuss Minimum supported versions of Node

### ISSUE THREAD

**Issue by Xiot** [uber/react-vis#1323]
Currently our CI tests run on Node 8 and 10.
The Node 8 tests were failing because the latest version of `mkdirp` (used by `storybook-addon-jsx` -> `js-beautify`) requires a minum of Node 10.

I've temporarily put in a `resolution` for `js-beatify` that keeps it pinned at a version that uses the old `mkdirp`.

If we remove the dependency on Node 8 we can remove this
https://github.com/uber/react-vis/blob/fa32e967a63924f19ff69643283c2e25e4a06fc1/package.json#L84

In terms of node versions, since the current LTS is `@12` I think we can safely remove `@8` from our CI, and probably add `12`

**Comment by a-peltier** [comment]
Agreed, I removed `@8` in `https://github.com/uber/react-vis/pull/1339`. 

Once it is merged I can add `@12` and remove the `resolution` field
